xds/src/main/java/io/grpc/xds/XdsLoadBalancerProvider.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2019 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.xds;
+
+import io.grpc.Internal;
+import io.grpc.LoadBalancer;
+import io.grpc.LoadBalancer.Helper;
+import io.grpc.LoadBalancerProvider;
+
+/**
+ * The provider for the "xds" balancing policy.  This class should not be directly referenced in
+ * code.  The policy should be accessed through {@link io.grpc.LoadBalancerRegistry#getProvider}
+ * with the name "xds" (currently "xds_experimental").
```

Comment:
Just curious what is the need to use "_experimental" suffix. Does it have anything to do with `io.grpc.LoadBalancer` being experimentalApi or the risk that existing policies/configs with "xds" will be processed by this code which is still "experimental"?

Reply:
The json schema of the service config is defined by the proto internally (as internally it's written as proto, then converted to and consumed as json). CL/230960643 changed the name from "xds" to "xds_experimental" and you can find rational in the discussions there.